lib/fastlane/plugin/revenuecat_internal/helper/update_hybrids_versions_file_helper.rb

```diff
@@ -35,13 +36,13 @@ def self.get_android_billing_client_version(android_version, github_token)
         matches[0]
       end
 
-      private_class_method def self.get_contents_file_github(file_path, repo_name, ref = 'main', github_token = nil)
+      private_class_method def self.get_contents_file_github(file_path, repo_name, ref = 'main', github_token)
         path = "/repos/revenuecat/#{repo_name}/contents/#{file_path}?ref=#{ref}"
-        response = Actions::GithubApiAction.run(server_url: 'https://api.github.com',
-                                                path: path,
-                                                http_method: 'GET',
-                                                body: {},
-                                                api_token: github_token)
+        response = Helper::GitHubHelper.github_api_call_with_retry(server_url: 'https://api.github.com',
```

Comment:
Move `require_relative 'github_helper'` to the start of this method so its not required while things are initializing but only when needed